wiki/Guides/Getting_Your_Map_Modded/zh.md

```diff
@@ -1,55 +1,73 @@
 
-**\#modreqs** 是一个在 osu! 中为找 mod 而设置的一个频道。只需要简单的输入 /np，然后坐和放宽即可。说不定会有 modder 甚至是 BN 过来 mod 你的图。（但 mod 与否取决于对方。）
+- **BN 请求 (BN req)：** 仅在[谱面审核成员 (BN)](/wiki/People/The_Team/Beatmap_Nominators) 开设的队列帖子或 `#modreqs` 频道里使用。当你认为普通的摸已经不足以帮助提升你的谱面，想要来自 BN 的更专业全面的摸时，可以向 BN 发出这类请求。
+- **歌曲元数据检查 (Meta check)：** 不言而喻，[摸图员](/wiki/Modding/Modder)也会检查谱面内的歌曲元数据是否正确。这也是常规摸的一部分流程内容。
+- **音效检查：** 摸图员会检查你的音效是否合曲。亦或是能否达到上架标准。
 
-[Bancho](/wiki/BanchoBot) 会截取在该频道中最后发布的 20 个谱面，以便 modder 找图进行 mod：
+## 摸图队列
 
-- 输入 `!request` 命令，BanchoBot 会随机发一个谱面链接给你。
-- 输入 `!request list` 命令，BanchoBot 会发给你最后5个发布的图，并附上链接。
-- 访问 [special map listing](https://osu.ppy.sh/p/beatmaplist&r=8)，将会将最后发布的20个谱面显示在这里。
+最广泛使用的找摸方法，就是去摸图队列里发帖。摸图队列帖子集中于[摸图队列论坛](https://osu.ppy.sh/community/forums/60)里，摸图员能通过开设帖子来快速提供摸图帮助，他人也可以在队列帖子里贴上自己的谱面来找摸。使用论坛找摸方便快捷————仅需完成以下几个步骤：
 
-如果你在短时间内连续发布三个以上的图，将会被 bancho 忽略掉。
+1. 前往[摸图队列论坛](https://osu.ppy.sh/community/forums/60)。
+2. 浏览论坛，找到和你谱面的游戏模式相同的摸图队列帖子。在贴上自己的谱面之前，请注意论坛是否标注为“开放 (open)”，否则你的请求大概率会被无视。
+3. 仔细阅读发帖规则。如果你的谱面没有达到规则里列出的标准，那很不幸，你只能去找其他的队列来贴上自己的谱面了。
+4. 仔细阅读摸图员设立的喜好（如果有）。投其所好，能节省所有人的时间。
+5. 点击屏幕下方`跳转到最后一条回复`（`>>`）按钮。
+6. 备注你的请求类型，并在`新回复`文本框内填好你的谱面链接，以及摸图员可能要求的额外谱面信息。
+7. 写完后，点击`发表`按钮。
 
-虽然收效甚微，不过这也是一个找 mod 的途径（并且发多少遍都行，不要担心）。
+一般来说，如果你想加快谱面被摸的进度，同时间在多个队列里发帖找摸也无可厚非。但记住，以上进程仅仅只能确保你发出了摸图请求，至于是否摸图还是取决于摸图员他们自己。
 
-## Modding Queues 版块
+同样值得注意的是，你最好在谱面几乎达到了上架标准后，再去请求 BN 摸图（如果他们特别标注允许接收新手谱师的请求，那么这些符合条件的新谱师可以试一试）。如果谱面与上架标准相差甚远，他们也可能会无视你的摸图请求。对于初学者谱师，还是建议先找足普通玩家的摸之后，再去询问 BN。
 
-[Modding Queues](https://osu.ppy.sh/community/forums/60) 会时不时有 modder 在这里开 queue。一般情况下，只要 queue 是开的 modder 都会接，然而你首先需要遵守 queue 中写明的特定的规则。
+## #modreqs 频道
 
-## 互相 mod (M4M)
+*警告：不要在频道里刷屏请求，否则你有可能被[禁言](/wiki/Silence)！建议在 `#modreqs` 频道的发言频率低于每小时一次。*
 
-跟“等价交换”差不多，用自己一个 mod 换一个别人的 mod。通常这个是最有效的找 mod 方式。甚至在自己在 \#modreq 发完之后等另外一个人发的时候马上就有一个潜在的 modder 在这里。互相 mod 的同时也在互相进步，这何乐而不为？
+`#modreqs` 频道位于 osu! 聊天区。玩家能在这里通过发送 `/np` 指令快捷加链接，并附上摸图请求的种类，从而为自己的谱面找摸。即使这种方法效率不高 摸图员或者 BN 也会抽空浏览这个频道，挑选一些心仪的谱面。
 
-## 直接找目标 modder
+`#modreqs` 频道已不再有以前的功能和用户量。但即便如此，这个频道也是找摸的好途径，值得一试。虽然频道里的摸多为单向摸，但也能见到[上面](#types-of-mods)所提到的其他请求类型。
 
-直接找 modder 也是一个找 mod 的方法。如果你发现某个玩家没开 queue 但有不少的 [kudosu](/wiki/Modding/Kudosu)（modder 的 kudosu 数量肯定不为零），你也不妨找一下这个玩家（论坛或游戏甚至是 QQ 直接私聊），只要不是很忙，基本上都会接的。当然，用这方法找泡甚至是找 rank 都行。以下是一些技巧：
+## 私聊其他玩家
 
-- 如果在签名或者 User page 已经**明确说明自己不接 mod**，那就不要劳烦他人了，不然可能会拉黑。
-- 避免开门见山，不要遇到其他玩家第一句话就是求 mod。尽可能随便聊一下，然后慢慢再达到你求 mod 的目的，这样效果会好很多。
-- 别人接了你的 mod 请求，就不要去催 mod 了。说不定别人压根就不喜欢这个谱面甚至是你个人，本来可以强行撑过去的被你这样一催，哪还有心情 mod 你的图？
+私聊其他玩家也是找摸的一种好方法。通常，如果摸图员没有开设摸图队列帖，那么可以去他们的个人主页查阅有关摸图的信息。阅读完毕后，即可私聊他们发出请求。虽然这些摸图员可能很难联系到，但他们或许有其他方法来提供摸图，例如架设一个私人摸图平台等等。
 
-## 华语圈找 mod 其它途径
+所有 BN 都在他们的个人主页上标注了摸图信息。不管他们有没有架设摸图队列帖，私聊他们是请求摸图，甚至请求提名谱面的重要方式。
 
-### Pending吧
+如果不在意摸的质量，可能最简单的找摸方法是：请求你的朋友来摸图。询问拥有作图经验的朋友，你大概率能收到满意的结果。
 
-传送门：[Link](https://tieba.baidu.com/f?kw=pending)
+## 华语圈找摸
```

Comment:
Note for maintainer: this section contains many Simplified Chinese community specific content.

Reply:
having community-specific info would in theory be fine, but directly linking to external groups could be questionable. this needs a few more opinions.

Reply:
deleted these links and keep the rest of description